frontend/public/components/dashboard/dashboards-page/overview-dashboard/inventory-card.tsx

```diff
@@ -73,7 +73,7 @@ const ClusterInventoryItem = withDashboardResources(
       let additionalResourcesLoadError = false;
       if (additionalResources) {
         additionalResourcesLoaded = additionalResources.every((r) =>
-          _.get(resources[r.prop], 'loaded'),
+          r.optional ? true : _.get(resources[r.prop], 'loaded'),
```

Comment:
Do we get a load error for optional resources that fail to load? I'd think we check `loaded || loadError` in the inventory card.